polaris-tokens/tsconfig.json

```diff
@@ -4,8 +4,8 @@
     "composite": true,
     "declarationDir": "dist/types",
     "emitDeclarationOnly": true,
-    "outDir": "dist",
-    "rootDir": "src"
+    "outDir": "dist/types",
+    "rootDir": "."
```

Comment:
I believe changing rootDir around messes with the folders structure generated within the outDir (I think it'll now have a src directory in it). If that's the case you'll need to update the `types` keys in package.json files